comp/core/tagger/taggerimpl/local/tagger.go

```diff
@@ -105,6 +106,20 @@ func (t *Tagger) Tag(entityID types.EntityID, cardinality types.TagCardinality)
 	return tags.Copy(), nil
 }
 
+// LegacyTag is an interface function that queries taggerclient singleton
```

Comment:
I am not sure if the comment here is still accurate.

After this change I think the tagger is no longer a global singleton, but rather a component.

Maybe it would be better to have the comment as:

```
// LegacyTag queries the tagger component for tags based on a string entity id.
```

WDYT?